src/Http/Middleware/ShareInertiaData.php

```diff
@@ -16,16 +16,19 @@ class ShareInertiaData
      */
     public function handle($request, $next)
     {
-        Inertia::share(array_filter([
-            'socialstream' => function () use ($request) {
-                return [
-                    'show' => Socialstream::show(),
-                    'providers' => Socialstream::providers(),
-                    'hasPassword' => ! is_null($request->user()->password),
-                    'connectedAccounts' => $request->user()->connectedAccounts,
-                ];
-            },
-        ]));
+        $user = $request->user();
+        if ($user) {
```

Comment:
@luiyongsheng these two lines above can be merged together like so:

```suggestion
        if ($user = $request->user()) {
```